README.md

```diff
@@ -15,7 +15,8 @@ This method is considered advanced and should only be used if one is an expert i
 Run as root (sudo su):
 
 ```bash
-curl -sL https://raw.githubusercontent.com/home-assistant/supervised-installer/master/installer.sh | bash -s
+wget https://raw.githubusercontent.com/home-assistant/supervised-installer/master/installer.sh
+bash installer.sh
```

Comment:
```suggestion
wget https://raw.githubusercontent.com/home-assistant/supervised-installer/master/installer.sh && bash installer.sh
```

If we want a one-liner

Reply:
But we should use `curl` 😉 
`wget` is not available on all minimal installations (and not a requirement for Supervised). In general, avoid `wget` in documentation.

Reply:
Yeah, I agree. Let's use `curl -Lo installer.sh https://raw.githubusercontent.com/home-assistant/supervised-installer/master/installer.sh`